_posts/2019-10-30-upgrade-reminder-zh.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+标题: 为比特币现金升级做好准备
```

Comment:
Same here, the words "title" and "subtitle" should be in English.

Reply:
good catch. thank you.